libs/community/langchain_community/agent_toolkits/openapi/planner.py

```diff
@@ -336,6 +345,7 @@ def create_openapi_agent(
     callback_manager: Optional[BaseCallbackManager] = None,
     verbose: bool = True,
     agent_executor_kwargs: Optional[Dict[str, Any]] = None,
+    allow_dangerous_requests: bool = False,
```

Comment:
Could you document the variable in args to explain the security concerns of allow dangerous requests?

This will help make sure that folks do not file CVEs against langchain code base

Reply:
@eyurtsev What's CVE?

Reply:
@eyurtsev Sure. I will use the description from `libs/community/langchain_community/tools/requests/tool.py`.
@AdamMomen I think it might mean Common Vulnerabilities and Exposures (CVE).